hawc_hal/healpix_handling/sparse_healpix.py

```diff
@@ -86,7 +117,7 @@ def as_dense(self):
         new_map = np.full(self.npix, self._fill_value)
 
         # Assign the active pixels their values
-        new_map[self._pixels_ids] = self._partial_map
+        new_map[self.pixels] = self._partial_map
```

Comment:
within a method of the class you can access private members (i.e., members starting with `_`). It is actually faster, because using properties have a slight overhead (although negligible most of the time).

Please change this line back to what it was:

`new_map[self._pixels_ids] = self._partial_map`